components/src/core/components/ListTable/ListTable.vue

```diff
@@ -226,26 +225,25 @@ export default defineComponent({
       },
     });
 
+    const computedHeaders = computed(() => {
+      if (props.loading && props.skeleton) {
+        return props.headers.filter(
+          header => !(header.name === 'action' && !header.title),
+        );
+      }
+      return props.headers;
+    });
     const cardHeaders = computed(() => {
-      const headers = props.headers.map(header => {
```

Comment:
```ts
const cardHeaders = computed(() => {
  if (props.loading && props.skeleton) {
    return [
      {
        name: "selector",
        cellType: "oxd-table-cell-checkbox",
      },
      ...props.headers.filter(
        (header) => header.name !== "action" && !header.title
      ),
    ];
  }
  if (props.selectable) {
    return [
      {
        name: "selector",
        cellType: "oxd-table-cell-checkbox",
      },
      ...headers,
    ];
  }

  return props.headers;
});

```

Consider this refactoring rather than using `computedHeaders`

Reply:
computedHeaders and cardHeaders has two different behaviors.